modules/multi-runner/outputs.tf

```
@@ -10,6 +10,7 @@ output "runners" {
     role_scale_up           = runner.role_scale_up
     role_scale_down         = runner.role_scale_down
     role_pool               = runner.role_pool
+    logfiles                = module.runners.logfiles
```

This won't work, you need to refer to the runner object.
```suggestion
    logfiles                = runner.logfiles
```

---

Thanks for the review @npalm . The PR got closed with an accidental force push, sincere apologies

---

I've also updated the PR description with the motivating use case(s) I'm trying to solve with this change

---

Another quick question: I was also planning on modifying the `multi-runner` module to allow `ami_id_ssm_parameter_name` as a parameter with `runner_config`. Should I raise a separate PR for it?

---

Yes please create a separate PR